subsys/usb/Kconfig

```diff
@@ -3,6 +3,12 @@
 # Copyright (c) 2016 Wind River Systems, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
+config USB
```

Comment:
nit: it'd probably be good to add this option back in the first commit, so that not commit is left in a non-buildable state. other than that, LGTM

Reply:
It allows me to test easier when it comes last, I just have to check out commit before that.

Reply:
I agree with @gmarull